src/VRDriver.hpp

```diff
@@ -52,6 +54,7 @@ namespace SlimeVRDriver {
         vr::HmdVector3_t GetPosition(vr::HmdMatrix34_t &matrix);
 
         bool sentHmdAddMessage = false;
+        Bridge *m_pBridge;
```

Comment:
Currently, we do not use (systems) hungarian notation in the driver. To be fair though, this is somewhat of a nitpick.

Reply:
```suggestion
        Bridge *bridge;
```
should've included the suggestion in the original comment.

I don't want to make suggestions for every occurrence, because basically every single variable/parameter that you added does not follow the style of the rest of the driver, and I don't want to spam the PR.